internal/store/pod.go

```diff
@@ -87,6 +87,34 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_pod_ready_time",
+			Type: metric.Gauge,
+			Help: "Unix timestamp for the moment a pod becomes ready.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				ms := []*metric.Metric{}
+
+				var readyTime *metav1.Time
+
+				for _, condition := range p.Status.Conditions {
+					if condition.Type == "Ready" {
```

Comment:
I forgot to check the condition status. The type might report as ready, but the pod might be not ready (thus reporting the status "False")